src/walk-path.js

```diff
@@ -30,18 +18,20 @@ class WalkPath extends React.Component {
     return (
       <div>
         <Polyline
-          className={className}
-          positions={this.props.positions} />
-        <Polyline
           className='hidden'
           positions={this.props.positions}
-          onMouseOver={() => this.handleMouseOver()}
-          onMouseOut={() => this.handleMouseOut()}
           onClick={e => {
             this.props.onClick();
             DomEvent.stopPropagation(e);
           }}
         />
+        <Polyline
+          className={className}
+          positions={this.props.positions} />
```

Comment:
There's a syntax error here: the element is closed before the `onClick` prop. But the `onClick` is not necessary if you add the `pointer-events` stuff.